packages/schematics/angular/ng-new/index.ts

```diff
@@ -60,11 +61,20 @@ export default function (options: NgNewOptions): Rule {
     zoneless: options.zoneless,
   };
 
+  const configOptions: ConfigOptions | undefined = options.aiConfig?.length
```

Comment:
Update this statement and remove in the below line;

```suggestion
  const configOptions: ConfigOptions | undefined = options.aiConfig && !options.aiConfig.includes(AiConfig.None)
```